### PR TITLE
refactor/1023

### LINKED ISSUE #1023: Make "openScope" the consistent capitalization throughout the repo
We have used various capitalizations in various places, recently we concluded `openScope` to be the most preferred form, so now we can make everything consistent.

### PR DESCRIPTION
**Pull request by erikquinn**
Resolves #1023.

Standardize `openScope` capitalization

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "openscope",
+  "name": "openScope",
```
Comment (n8rzz) [inline app.json]: this should stay lowercase as I believe this maps directly back to our app name on heroku

File: package-lock.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "openscope",
+  "name": "openScope",
   "version": "6.3.0-BETA",
```
Comment (n8rzz) [inline package-lock.json]: you should never directly modify this file.  the `package-lock.json` file is generated after an `npm` install.  I'd recommend deleting this file and re-running `npm install`.  That should also result in an update of the version number, which is also wrong here.

File: src/templates/layout.hbs [hunk 3]
```diff
@@ -7,14 +7,14 @@
 <html>
     <head>
         <meta charset="utf-8"/>
-        <meta property="Description" content="A free and open source Air Traffic Control Simulator."/>
-        <meta property="og:title" content="Openscope - Air Traffic Control Simulator"/>
-        <meta property="og:description" content="A free and open source Air Traffic Control Simulator."/>
+        <meta property="Description" content="Free and open source air traffic control simulator"/>
+        <meta property="og:title" content="openScope Air Traffic Control Simulator"/>
```
Comment (n8rzz) [inline src/templates/layout.hbs]: `Air Traffic Control Simulator` is cased differently here, is that on purpose?

File: src/templates/layout.hbs [hunk 4]
```diff
@@ -34,7 +34,7 @@
         <link rel="icon" type="image/png" sizes="96x96" href="/assets/images/favicon/favicon-96x96.png">
         <link rel="icon" type="image/png" sizes="16x16" href="/assets/images/favicon/favicon-16x16.png">
 
-        <title>Openscope - Air Traffic Control Simulator</title>
+        <title>openScope Air Traffic Control Simulator</title>
```
Comment (n8rzz) [inline src/templates/layout.hbs]: `Air Traffic Control Simulator` is cased differently here, is that on purpose?

Comment (erikquinn) [reply src/templates/layout.hbs]: I don't quite follow-- different from what?

Comment (n8rzz) [reply src/templates/layout.hbs]: different from lines 10 or 12 where it is all lowercase

File: src/templates/layout.hbs (hunk 3 above)
Comment (erikquinn) [reply src/templates/layout.hbs]: I see what you're saying:
```
<meta property="Description" content="Free and open source air traffic control simulator"/>
<meta property="og:title" content="openScope Air Traffic Control Simulator"/>
<meta property="og:description" content="Free and open source air traffic control simulator"/>
```

They were cased based on which property they were under.
- I assume description to be somehow used to describe what the site is, so I used normal sentence-style capitalization.
- Title was cased like the title of a book

That was the thought process anwyay.

File: src/templates/layout.hbs (hunk 4 above)
Comment (erikquinn) [reply src/templates/layout.hbs]: I see what you're saying:
```
<meta property="Description" content="Free and open source air traffic control simulator"/>
<meta property="og:title" content="openScope Air Traffic Control Simulator"/>
<meta property="og:description" content="Free and open source air traffic control simulator"/>
```

They were cased based on which property they were under.
- I assume description to be somehow used to describe what the site is, so I used normal sentence-style capitalization.
- Title was cased like the title of a book

That was the thought process anwyay.

File: src/templates/layout.hbs (hunk 3 above)
Comment (erikquinn) [reply src/templates/layout.hbs]: https://github.com/openscope/openscope/pull/1024#discussion_r204261319

Responded to in the next comment section on `layout.hbs` below.

File: package-lock.json (hunk 2 above)
Comment (erikquinn) [reply package-lock.json]: done

**Pull request merged**